backend/Dockerfile

```diff
@@ -1,29 +1,31 @@
-FROM debian:stretch-slim
+FROM golang:1 as builder
+
+COPY build/*.go build/go.* /src/main/
+COPY *.go go.* /backend/
+COPY cothority/ /backend/cothority/
+COPY proto/ /backend/proto/
+COPY stainless/ /backend/stainless/
+
+RUN cd /src/main && go build -v -o /conode
+
+FROM debian:bookworm-slim as runner
 
-# See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863199
-RUN mkdir -p /usr/share/man/man1
 # libgomp is needed for z3
-RUN apt-get update; \
-    apt-get install --no-install-recommends --yes \
-    procps \
-    openjdk-8-jre-headless \
-    libgomp1 \
-    unzip \
-    ; apt-get clean
-# Backports needed for 'npm'...
-RUN echo 'deb http://deb.debian.org/debian stretch-backports main' >> /etc/apt/sources.list
-RUN apt-get update; \
-    apt-get install --target-release stretch-backports --no-install-recommends --yes \
-    npm \
-    ; apt-get clean
-RUN npm install --global solc@0.5.10
-
-COPY build/conode.Linux.x86_64 /usr/local/bin/conode
+RUN apt update \
+    && apt install --no-install-recommends --yes \
```

Comment:
Nitpick: I like my `&&` at the end of the line to have all the commands aligned. But I'm sure you have a good reason to put it at the beginning of the line - why?